modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -0,0 +1,233 @@
+= Running entitled builds using SharedSecret objects
+
+[role="_abstract"]
+Configure and perform a build in one namespace that securely uses RHEL entitlements from a `Secret` object in another namespace.
+
+Previously, {product-title} 4.9.x and earlier suggested accessing RHEL entitlements from OpenShift Builds by creating a `Secret` object with your subscription credentials in the same namespace as your `Build` object. Now, with {product-title} 4.10 and later, you can access your credentials and certificates from a `Secret` object in one of the {product-title} system namespaces. You run entitled builds with CSI volume mount of a `SharedSecret` custom resource (CR) instance that references the `Secret` object.
+
+This procedure relies on the newly introduced The Shared Resources CSI Driver feature, which you can use to declare CSI Volume mounts in {product-title} Builds. It also relies on the {product-title} Insights Operator.
+
+:FeatureName: The Shared Resources CSI Driver
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+:FeatureName: TBD
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+The Shared Resources CSI Driver and TBD features also belong to the `TechPreviewNoUpgrade` feature set, which is a subset of the current Technology Preview features. You can enable the `TechPreviewNoUpgrade` feature set on test clusters, where you can fully test them while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents updates. This feature set is not recommended on production clusters. See "Enabling Technology Preview features using feature gates" in the following "Additional resources" section.
+====
+
+.User roles
+
+The following procedure includes several types of users:
+
+* An _administrator_ with cluster-level permissions to access multiple namespaces, create cluster-scoped objects, and create `Role` and `RoleBinding` objects around those cluster-scoped objects, both for discovery and use.
+* Optionally, _namespace administrators_ can perform some of the tasks in this example, such as creating  `Role` and `RoleBinding` objects for users or service accounts.
+* General users of the cluster, such as  _developers_ or _testers_, are generally restricted to access a single namespace to build images with OpenShift Builds.
++
+So that these users can use RHEL entitlements in their builds, administrators give them access to discover `SharedSecret` and `SharedConfigMaps` custom resource (CR) instances that they can reference in their `BuildConfig` objects. Also, their namespace `builder` service account has been permitted to mount the specific `SharedSecret` CR instance as a Shared Resource CSI Driver volume in OpenShift Build pods.
+
+.Prerequisites
+
+* You have enabled the  `TechPreviewNoUpgrade` feature set by using the feature gates.
+* You have created and registered a `Secret` object at https://console.redhat.com/.
```

Comment:
Yeah the blog post has 

```
A SharedSecret that references the Secret where the Insights Operator stores the Subscription Credentials available to the cluster created and registered at https://console.redhat.com/
```

Did you try to "simplify" things?  If so, you've gone to the point where what you have here is inaccurate / misleading.